src/reducers/postReducer.test.js

```diff
@@ -0,0 +1,57 @@
+import { addPost, deletePost, updatePost } from '../actions/blogActions';
+import reducer from './postReducer';
+
+describe('post reducer', () => {
+  it('handles the ADD_POST action', () => {
+    const state = [];
+    const action = addPost({
+      id: 6,
+      title: chance.name(),
```

Comment:
I don't see that you're importing the chance library. Your test is only passing right now because you're expecting any string including an empty string.